index.html

```diff
@@ -110,22 +118,66 @@ <h4 class="backyear">Backend Dev</h4>
           <h4 class="backyear">2015</h4>
         </div>
         <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
         </p>
         <ul class="tags">
           <li class="tag">html</li>
           <li class="tag">css</li>
           <li class="tag">javascript</li>
         </ul>
         <div class="action">
-          <button type="button">See Project</button>
+          <button data-id="1" type="button" id="tonic-modal-open">See Project</button>
+        </div>
+      </div>
+
+      <!-- Tonic card modal -->
+      <div class="tonic-card-modal-container" id="tonic-card-modal-container">
+        <div class="tonic-card-modal">
+          <h1 class="tonic-title">Tonic Modal</h1>
+          <div class="tonic-stacks">
+            <ul>
+              <li class="tonic-stack1"></li>
+              <li class="tonic-stack2"></li>
+              <li class="tonic-stack3"></li>
+            </ul>
+          </div>
```

Comment:
- [ ] Please make sure the popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colours, images, tests, margins).
- Your design
![screenshot (2)](https://user-images.githubusercontent.com/44624138/141211621-dd766b10-0a7d-4317-a4d8-9d2e3f4cbe56.png)
- Figma design
![screenshot (3)](https://user-images.githubusercontent.com/44624138/141211638-49fb2cce-9075-413b-8864-ad1099e430fb.png)